src/Components/README.md

```diff
@@ -163,3 +163,22 @@ builder.AddAzureServiceBus(settings =>
 ```
 
 - If both secret and passwordless mechanisms are configured, the secret credential overrides the passwordless identity setting.
+
+## Checklist for new Components
+
+New Components MUST have:
+
+* README.md
+* ConfigurationSchema.json file
+* Public APIs
+    * Extension method off IHostApplicationBuilder and Settings object
+* Tests
+    * [ConformanceTests](../../tests/Aspire.Components.Common.Tests/ConformanceTests.cs)
+    * Other unit tests as needed
+* Tracing support
+
+New Components SHOULD have:
+
+* Logging support
+* Metrics support
+* Health Checks
```

Comment:
This should be required as well.

Reply:
😢

Reply:
I added a line that says components need to have justification when they don't include these.